mods/ra/maps/sarin-gas-1-crackdown/crackdown.lua

```diff
@@ -42,27 +42,21 @@ SetupTriggers = function()
 		greece.MarkCompletedObjective(objDestroyAllTrucks)
 	end)
 
-	Trigger.OnEnteredFootprint({ TruckEscapeCenter.Location }, function(actor, triggerlose1)
-		if actor.Owner == ussr and actor.Type == "truk" then
-			Trigger.RemoveProximityTrigger(triggerlose1)
-			actor.Destroy()
-			greece.MarkFailedObjective(objDestroyAllTrucks)
-		end
-	end)
-
-	Trigger.OnEnteredFootprint({ EscapeNorth10.Location }, function(actor, triggerlose2)
-		if actor.Owner == ussr and actor.Type == "truk" then
-			Trigger.RemoveProximityTrigger(triggerlose2)
-			actor.Destroy()
-			greece.MarkFailedObjective(objDestroyAllTrucks)
-		end
-	end)
+	SetTrukTrigger(TruckEscapeCenter.Location, triggerlose1)
+	SetTrukTrigger(EscapeNorth10.Location, triggerlose2)
+	SetTrukTrigger(EscapeSouth5.Location, triggerlose3)
+end
 
-	Trigger.OnEnteredFootprint({ EscapeSouth5.Location }, function(actor, triggerlose3)
+SetTrukTrigger = function(location, trigger)
```

Comment:
The `trigger` argument here can be removed. (It is unused, since the function declared in `OnEnteredFootprint` overwrites it with its own parameter definition.)